packages/components/Shimmer/src/Shimmer.tsx

```diff
@@ -0,0 +1,105 @@
+/** @jsx withSlots */
+import {
+  compose,
+  UseSlots,
+  buildProps,
+  mergeProps,
+  withSlots
+} from '@fluentui-react-native/experimental-framework';
+import { requireNativeComponent } from 'react-native';
+import { IViewProps } from '@fluentui-react-native/adapters';
+import * as React from 'react';
+
+const shimmerName = "Shimmer"
+
+const NativeShimmerView = requireNativeComponent('MSFShimmerView');
+
+export interface IShimmerViewAppearance {
+  /**
+   * Color to tint the shimmer boxes. Defaults to the Fabric default color.
+   */
+  tintColor: string;
+
+  /**
+   * Corner radius on each view.
+   */
+  cornerRadius: number;
+
+  /**
+   * Corner radius on each UILabel. Set to < 0 to disable and use default `cornerRadius`.
+   */
+  labelCornerRadius: number;
+
+  /**
+   * usesTextHeightForLabels: True to enable shimmers to auto-adjust to font height for a UILabel -- this
+   * will more accurately reflect the text in the label rect rather than using the bounding box. `labelHeight`
+   * will take precendence over this property.
+   */
+  usesTextHeightForLabels: boolean;
+
+  /**
+   * If greater than 0, a fixed height to use for all UILabels. This will take precedence
+   * over `usesTextHeightForLabels`. Set to less than 0 to disable.
+   */
+  labelHeight: number;
+}
+
+export interface IShimmerAppearance {
```

Comment:
Does setting all of these work? I probably missed it but didn't see where we propagate these to the swift FUA control

Reply:
updateShimmerAppearance() inside ShimmerContainerView.swift